test/Integration/broadcast-2d.mlir

```diff
@@ -0,0 +1,27 @@
+// RUN: tpp-run --vector-to-XSMM %s -e columnBroadcast --entry-point-result=void -print --seed 123 2>&1 | FileCheck %s -check-prefix=COLUMNBROADCAST
+// RUN: tpp-run --linalg-to-loops %s -e columnBroadcast --entry-point-result=void -print -seed 123 2>&1 | FileCheck %s -check-prefix=COLUMNBROADCAST
+
+
+#map2 = affine_map<(d0, d1, d2) -> (d2)>
+#map3 = affine_map<(d0, d1, d2) -> (d0, d1, d2)>
+
+func.func @columnBroadcast(%arg0: tensor<8xf32>, %arg1: tensor<2x4x8xf32>) -> tensor<2x4x8xf32> {
```

Comment:
Is this really a 2D column broadcast?

Reply:
It is a column broadcast, which is what the function is called, the file should probably be renamed broadcast with 2D destination or something, can you please suggest an appropriate name for the test file?